Package.swift

```diff
@@ -24,7 +24,8 @@ let package = Package(
     targets: [
         .target(
             name: "BlueprintUI",
-            path: "BlueprintUI/Sources"
+            path: "BlueprintUI/Sources",
+            swiftSettings: [.unsafeFlags(["-enable-testing"])]
```

Comment:
Oops, meant to leave a comment on this. Since we don't publish this package, it lets us run our tests in `Release` mode (eg, allows `@testable` imports). Can remove, tho it means it has to be added back when doing any perf testing.

Reply:
Removing; can't figure out a good way to make this optional otherwise